newIDE/app/scripts/create-new-theme.js

```diff
@@ -0,0 +1,70 @@
+let args = process.argv;
+let path = require('path');
+let fs = require('fs');
+
+let theme = args[2];
+if (!theme) {
+  console.error('Please enter a theme name');
+  process.exit(1);
+}
+
+if (!theme.toLowerCase().endsWith('theme')) {
+  theme = theme + 'Theme';
+}
+
+const themeId = theme.replace(/\s+/, '');
+
+
+const dir = path.resolve(__dirname, '../src/UI/Theme/', themeId);
+if (fs.existsSync(dir)) {
+  console.warn('Theme `' + theme + '` already exists');
+  process.exit(0);
+}
+
+// Create Theme Folder
+fs.mkdirSync(dir);
+console.log('Created Folder', dir);
+
+// Create Style Library
+console.log(path.resolve(__dirname, './theme-templates/theme.json'));
+fs.writeFileSync(
+  path.resolve(dir, 'theme.json'),
+  fs.readFileSync(path.resolve(__dirname, './theme-templates/theme.json')).toString(),
+);
+console.log('Created theme.json');
+
+// Create Theme Object
+fs.writeFileSync(
+  path.resolve(dir, 'index.js'),
+  fs.readFileSync(path.resolve(__dirname, './theme-templates/index.js')).toString().replace(/\$themeId/g, themeId),
```

Comment:
two nitpicking:

* this is a single "template", so the folder should I think be called "theme-template" or even "new-theme-template" for extra explicitness.
* I usually prefer to make files were replacement is done extra visible by using SCREAMING_CASE for the replacements. i.e here: $THEME_ID

Reply:
That's much easier to see! done ✅ 
Also, there's two files in that folder. One's a template for the `.js` file, and there's another for a default `theme.json`